src/renderer/modals/Send/steps/StepAmount.js

```diff
@@ -30,7 +30,7 @@ const StepAmount = ({
 
   return (
     <Box flow={4}>
-      <TrackPage category="Send Flow" name="Step Amount" />
+      <TrackPage category="Send Flow" name="Step 2" />
```

Comment:
nope we must not change these. totally breaking change for data team. especially not with numbers.